pkg/model/crd.go

```diff
@@ -387,6 +387,34 @@ func (r *CRD) GetOutputWrapperFieldPath(
 	return &opConfig.OutputWrapperFieldPath
 }
 
+// OutputShapeName returns name of the Output shape for given operation.
```

Comment:
mismatched function name here... plus, I agree with @A-Hilaly about making the function return a Shape instead of a string name. Maybe call this method `GetOutputShape()`?

Reply:
Updated code.